internal/common/helpers.go

```diff
@@ -12,3 +12,13 @@ func TrimStringSlice(slice []string) {
 		slice[iterator] = strings.TrimSpace(item)
 	}
 }
+
+// StringSliceContains checks if the slice contains the given string.
+func StringSliceContains(slice []string, s string) bool {
```

Comment:
nit: I'm wondering to how many places in the source code it can applied :)

Reply:
We can look for them in a follow up, and perhaps if we wait to https://tip.golang.org/doc/go1.18#generics we can make it generic and apply it even more extensively :)